.github/workflows/playground.yaml

```diff
@@ -32,7 +32,7 @@ jobs:
       - uses: jetli/wasm-pack-action@v0.4.0
       - uses: jetli/wasm-bindgen-action@v0.2.0
       - name: "Run wasm-pack"
-        run: wasm-pack build --target web --out-dir playground/src/pkg . -- -p ruff
+        run: wasm-pack build --target web --out-dir ../../playground/src/pkg crates/ruff --out-name=ruff
```

Comment:
I was confused by this `../../` path, but I guess it's relative to `crates/ruff`? Seems to work!

Reply:
Yeah, that's confusing. It seems that `wasm-pack` interprets the out dir path relative to the build target...